client/components/title-format-editor/index.jsx

```diff
@@ -94,21 +95,10 @@ if ( ! Array.prototype.fill ) {
 
 // The below is a `require()` statement becuase it needs to
 // be loaded in after the polyfills are created.
-const {
-	CompositeDecorator,
-	Editor,
-	EditorState,
-	Entity,
-	Modifier,
-	SelectionState,
-} = require( 'draft-js' );
+import { CompositeDecorator, Editor, EditorState, Entity, Modifier, SelectionState } from 'draft-js';
```

Comment:
Hey @ryelle. This specific change doesn't seem to take into account the comment just above, which is probably why loads of errors are thrown in the browser's console when loading the [`Traffic` page](https://wordpress.com/settings/traffic):

<img width="781" alt="screenshot" src="https://user-images.githubusercontent.com/594356/32230980-aa258aea-be54-11e7-8b59-7672468348f8.png">

Could you have a look when you have a few?

Reply:
I switched it back locally and am still seeing these errors – I think because we check `if ( ! Array.prototype.fill ) {` before defining the polyfill, and `Array.prototype.fill` exists (it's supported in this browser and then we wrap it in the warning in `client/lib/wrap-es6-functions`). Are you sure this worked correctly before this PR merged?

Reply:
Maybe related: #19177 - `draft-js` just received a version bump. Worth checking before that PR was merged.